frontend/web/components/IdentitiesPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useEffect, useState } from 'react';
+import axios from 'axios';
+import { Bar } from 'react-chartjs-2';
+import { Chart as ChartJS, CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend } from 'chart.js';
+import PageTitle from './PageTitle';
+import '../App.css'; // Adjust the path if necessary
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend);
+
+type IdentitiesPageType = {
+  match: {
+    params: {
+      environmentId: string;
+      projectId: string;
+    };
+  };
+};
+
+const IdentitiesPage: FC<IdentitiesPageType> = ({
+  match: {
+    params: { environmentId, projectId },
+  },
+}) => {
+  const [identitiesData, setIdentitiesData] = useState<{ identities: number; day: string }[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
```

Comment:
This should ideally be rtk, or at lest fetch / _data.get

Reply:
This api doesn't track new identities, but rather calls to identify with new or existing ids